designs/comprehensive-checks.stanza

```diff
@@ -187,7 +187,7 @@ pcb-component crystal-resonator :
 pcb-module resonator-module :
   public inst X1 : crystal-resonator
   public inst C1 : ceramic-capacitor
-  property(C1.tolerance) = [0.99, 1.01]
+  property(C1.tolerance) = Toleranced(0.99, 1.0, 1.01)
```

Comment:
The actual values within Toleranced match a proposal discussed in JITX-5427.

I've gone with the `min/typ/max` proposed there - but, can update here to match whatever is decided.